R/pkg/R/DataFrame.R

```diff
@@ -212,9 +212,9 @@ setMethod("showDF",
 
 #' show
 #'
-#' Print the SparkDataFrame column names and types
+#' Print class and type information of a SparkR object.
 #'
-#' @param object a SparkDataFrame.
+#' @param object a SparkR object. Can be a SparkDataFrame, Column, GroupedData, WindowSpec.
```

Comment:
ok this is the case the existing generic from method is used.
let's leave it then.